fs/expose/dokan/__init__.py

```diff
@@ -880,26 +958,29 @@ def mount(fs, drive, foreground=False, ready_callback=None, unmount_callback=Non
         * numthreads:  number of threads to use for handling Dokan requests
         * fsname:  name to display in explorer etc
         * flags:   DOKAN_OPTIONS bitmask
+        * securityfolder:  folder path used to duplicate security rights on all folders 
         * FSOperationsClass:  custom FSOperations subclass to use
 
     """
     if libdokan is None:
         raise OSError("the dokan library is not available")
-    drive = _normalise_drive_string(drive)
+    _check_path_string(path)
     #  This function captures the logic of checking whether the Dokan mount
     #  is up and running.  Unfortunately I can't find a way to get this
-    #  via a callback in the Dokan API.  Instead we just check for the drive
+    #  via a callback in the Dokan API.  Instead we just check for the path
     #  in a loop, polling the mount proc to make sure it hasn't died.
+
     def check_alive(mp):
-        if mp and mp.poll() != None:
+        if mp and mp.poll() is not None:
             raise OSError("dokan mount process exited prematurely")
+
     def check_ready(mp=None):
         if ready_callback is not False:
```

Comment:
Hmm, I guess this should actually be `if ready_callback is not None:` ?

(and I wonder if the `opts["ready_callback"] = False` later on should also be using `None`? Or maybe there's some horrible subtlety here that I'm not fully understanding? :-S )